TaroEld/BBbuilder
# Add ES6 support

Add ES6 support by using transpilation.

How to use/test :
- run cli `bbbuilder.exe build [path] -transpile`
- if nodejs isn't installed on your computer, it will show an error with a link to install nodejs. Install it and do the command again.
- mod should be ready in data game folder. Check that the js files don't contain es6 syntax at that point (class for example). If not, the transpilation work without issue.
